### PR TITLE
[doc] amend DUA and MLR fast poll comments

### PR DESCRIPTION
**Pull request by librasungirl**

### CODE REVIEW

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

thanks

File: src/core/thread/dua_manager.cpp [hunk 1]
```diff
@@ -508,7 +508,8 @@ void DuaManager::PerformNextRegistration(void)
     mIsDuaPending   = true;
     mRegisteringDua = dua;
 
-    // TODO: (DUA) need update when CSL is enabled.
+    // Regardless of csl enabled or not, sleepy end device should always rely on
+    // fast data poll to fetch the response timely when the parent is of version 1.1.
```
Comment (jwhui) [inline src/core/thread/dua_manager.cpp:512]: This comment doesn't seem clear to me. CSL cannot be used when parent "is of version 1.1". Can we make this more clear?

Comment (librasungirl) [reply src/core/thread/dua_manager.cpp]: Updated. Please take another review.

Comment (jwhui) [reply src/core/thread/dua_manager.cpp]: Much more clear. Thanks!

**Review by jwhui — APPROVED** [review]

**Pull request merged**